pyeudiw/satosa/default/openid4vp_backend.py

```diff
@@ -0,0 +1,327 @@
+import uuid
+import json
+from typing import Callable
+from pyeudiw.jwk import JWK
+from satosa.context import Context
+from pydantic import ValidationError
+from ..impl.trust import BackendTrust
```

Comment:
please do not use relative imports, like .. or . 

we use absolute imports only

Reply:
please change the name impl to something more eloquent

what's the purpose of the word `impl`?